### DISCUSSION TITLE
Showing media feature values

### DISCUSSION THREAD

**Discussion by wbamberg** [mdn/mdn-community#135] [Platform] [answered]
@NiedziolkaMichal filed a new PR yesterday: https://github.com/mdn/yari/pull/6566. I haven't tried it out yet but AFAICT it adds a box to media feature pages, showing the current value of the feature on the user's device. It's implemented as a KS macro with some Yari magic behind it.

Without getting into the implementation, I'd like to know whether people think this is a feature we want to have in our pages. At a glance it seems like a nice thing to have, but I might have missed something.

**Comment by teoli2003** [comment]
I find this very useful. It doesn't add too much cruft to the page and gives something to play with it interactively.

**Comment by bsmth** [comment, marked as answer]
From the linked PR:

>the additional complexity outweighs the user benefit of this nice-to-have feature, so we're closing the PR without merge.

So I'm going to close with that as a resolution. Thanks :)
